src/Command/Dev/UpdateComposer/clear_module_requirements.php.tpl

```diff
@@ -0,0 +1,36 @@
+<?php
+
+$composer = json_decode(file_get_contents(__DIR__ . '/composer.json'), true);
+
+if (isset($composer['repos'])) {
+    $repos = array_filter($composer['repos'], function($val) {
+       return isset($val['type']) && $val['type'] == 'path';
+    });
+} else {
+    $repos = [];
+}
+$packages = array_keys($repos);
+
+function clearRequirements($dir, $packages) {
+    if (!file_exists($dir . '/composer.json')) {
+        return;
+    }
+
+    $composerJson = json_decode(file_get_contents($dir . '/composer.json'), true);
+
+    foreach ($composerJson['require'] as $requireName => $requireVersion) {
+        if (in_array($requireName, $packages)) {
+            $composerJson['require'][$requireName] = '*@dev';
```

Comment:
Replaced `unset` with just reseting the version and allowing `dev` dependencies. This would help detect problems with dependencies in development stage, while avoid problems with versioning.